### PR TITLE
ecma: highlight private properties and methods

### PR DESCRIPTION
**Pull request by numToStr**
Preview:
image

### CODE REVIEW

File: queries/ecma/highlights.scm [hunk 1]
```diff
@@ -87,6 +90,9 @@
 (call_expression
   function: (member_expression
     property: (property_identifier) @method))
+(call_expression
+  function: (member_expression
+    property: (private_property_identifier) @method))
```
Comment (numToStr) [inline queries/ecma/highlights.scm:95]: I was wondering though. Is it possible to merge this into one? Like
```scheme
(call_expression
  function: (member_expression
    property: (property_identifier|private_property_identifier) @method))
```

Comment (theHamsta) [reply queries/ecma/highlights.scm:95]:
```
(call_expression
  function: (member_expression
    property: [(property_identifier) (private_property_identifier)] @method))
```

Comment (numToStr) [reply queries/ecma/highlights.scm:95]: Thanks @theHamsta. I'll update the PR shortly.

Comment (theHamsta) [reply queries/ecma/highlights.scm:95]: We just added highlight tests to this repo. If you want to you can submit your example file with highlight assertions for the change (e.g. private fields that are now highlighted)

Format would be to add a caret below pointing the highlight that you changed
```ts
class {
  #priv_field = "World";
//   ^ property
}

```

This is not a must. I'm just  Your PR is awesome already!

Comment (numToStr) [reply queries/ecma/highlights.scm:95]: Sure, why not. I'll try to add test for this PR :)

**Comment by numToStr** [comment]
@theHamsta I've added the test. I think now it is good to go.

(I can't believe tests got passed in the first try 😄)

**Review by theHamsta — APPROVED** [review]
Nice! Thank you!

**Pull request merged**